### ISSUE TITLE
Received true for a non-boolean attribute rotate.

### ISSUE THREAD

**Issue by rcwell** [sanusart/react-dropdown-select#86]
Hi, just stumbled upon a little bug. Was just following from examples and upon checking console, there is a warning

> Warning: Received `true` for a non-boolean attribute `rotate`. if you want to write it to the DOM, pass a string instead: rotate="true" or rotate={value.toString()}.

Here is my current code:

```javascript
<Select
    values={values}
    options={options}
    onChange={setValues}
    labelField={'name'}
    valueField={'id'}
/>
```

**Comment by codewhynot** [comment]
the same problem

**Comment by sanusart** [comment]
Thanks fixed in version: 4.3.1
